easybuild/easyconfigs/i/intel/intel-2020.00.eb

```diff
@@ -10,11 +10,12 @@ toolchain = SYSTEM
 
 local_compver = '2020.0.166'
 local_gccver = '9.2.0'
+local_gccsuff = '-GCC-%s' % (local_gccver)
 dependencies = [
     ('GCCcore', local_gccver),
     ('binutils', '2.32', '-GCCcore-%s' % local_gccver),
```

Comment:
Ehhh, there is no binutils-2.32 with a "-GCCcore-9.2.0" versionsuffix.
Shouldn't this (and also intel/2020a) be
('binutils', '2.32', '', ('GCCcore'. local_gccver)),
Just like in GCC-9.3.0.eb?

@boegel ??

Reply:
`binutils-2.32-GCCcore-9.2.0.eb` is there, you need more coffee ;)

Reply:
Oh, wait, I see what you mean... Yeah, should be `('binutils', '2.32', '', ('GCCcore'. local_gccver)),`.

So, I need more coffee.

Reply:
fixed in wrapper PR #10083